source/ember-inspector/data.md

```diff
@@ -1,40 +1,33 @@
-If you are using a data library that supports the Ember Inspector such
-as Ember Data, you can inspect your models by clicking on the `Data` menu.
+If you are using Ember Data, you can inspect your models by clicking on the `Data` menu.
 
 When you open the Data tab, you will see a list of model types defined
-in your application. Each model type has the number loaded records next
-to it. You can click on one type to view all of its loaded records.
+in your application, along with the number of loaded records.
+Click on a model type, and the Inspector displays those records.
 
 <img src="../../images/guides/ember-inspector/data-screenshot.png" width="680"/>
 
 ### Inspecting Records
 
-Each row in the list represents a record, with the first four attributes
-as columns (just enough attributes to identify the record). To view all
-of the attributes and more properties, click on the record and it will
-open in the object inspector.
+Each row in the list corresponds to one record. The first four model attributes are shown in the list view. Clicking on the record will open the Object Inspector for that record, and display all attributes.
 
 <img src="../../images/guides/ember-inspector/data-object-inspector.png"
 width="680"/>
 
 ### Record States and Filtering
 
-The data tab is a real time representation of your application's store.
-Any changes such as added records, modifications, or deleted records will
-immediately be reflected in the data tab. Record attributes are also
-bound.
-
-New unsaved records are green.
+The Data tab is kept in sync with the data loaded in your application.
+Any record additions, deletions, or changes are reflected immediately. If you have unsaved
+records, they will be displayed in green.
 
 <img src="../../images/guides/ember-inspector/data-new-records.png"
 width="680"/>
 
-Modified unsaved records are blue.
+Unsaved record modifications are in blue.
 
 <img src="../../images/guides/ember-inspector/data-modified-records.png"
 width="680"/>
 
-You can filter records based on state by clicking on one of the pills
+You can filter the list view based on record state by clicking on one of the pills
```

Comment:
I don't like "the list view" here because it sounds too close to list-view. "State" is also kind weird, maybe "You can filter by New, Modified, Clean by clicking the respective pill" or something, maybe link to an explanation somewhere of what each state means?